terra_notebook_utils/drs.py

```diff
@@ -121,6 +127,56 @@ def copy_to_local(drs_url: str,
         logger.info(f"Downloading {drs_url} to {filepath}")
         blob.download_to_file(fh)
 
+def print_bytes(source: Callable, num_bytes: int, buffer: int = MULTIPART_THRESHOLD):
```

Comment:
Consider passing in typing.IO instead of typing.Callable. This will allow you to use standard io `read`, and avoids the chunking arithmetic. It also avoids the implicit requirements on `source`, which must support the `start` and `end` parameters.

You can pass in a readable stream over a GS blob with:
```
with gscio.Reader(blob) as handle:
    print_bytes(handle, num_bytes, buffer)
```